app/src/main/java/com/framgia/fsalon/screen/scheduler/detail/BookingDetailPresenter.java

```diff
@@ -7,6 +7,7 @@
 import io.reactivex.annotations.NonNull;
 import io.reactivex.disposables.CompositeDisposable;
 import io.reactivex.disposables.Disposable;
+import io.reactivex.functions.Consumer;
```

Comment:
ERROR: Unused import: io.reactivex.functions.Consumer. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports